pylint_requests/utils.py

```diff
@@ -59,10 +60,13 @@ def _lookup(node: NodeNG) -> List[NodeNG]:
         try:
             obj = next(node.expr.infer())
         except InferenceError:
-            pass
-        else:
-            if isinstance(obj, bases.Instance):
+            return []
+
+        if isinstance(obj, bases.Instance):
+            try:
                 return obj.getattr(node.attrname)
+            except AttributeInferenceError:

```

Comment:
Is `AttributeInferenceError` a subclass of `InferenceError`? If so, I would prefer to catch `InferenceError` here just in case.

Reply:
The `AttributeInferenceError` and `InferenceError` both extend `ResolveError`. 'm not shure in which situations we might catch another child of `ResolveError` so I chose the most specific error class.
```
(Pdb) e
AttributeInferenceError('{attribute!r} not found on {target!r}.')
(Pdb) e.__class__
<class 'astroid.exceptions.AttributeInferenceError'>
(Pdb) e.__class__.__bases__
(<class 'astroid.exceptions.ResolveError'>,)
(Pdb) e.__class__.__bases__[0].__bases__
(<class 'astroid.exceptions.AstroidError'>,)
(Pdb) e.__class__.__bases__[0].__bases__[0].__bases__
(<class 'Exception'>,)
```

<img width="509" alt="image" src="https://user-images.githubusercontent.com/3585469/81012575-73dca280-8e62-11ea-9a71-28a34108760a.png">